CHANGELOG_PENDING.md

```diff
@@ -1,5 +1,8 @@
 ### Improvements
 
+- Fix `pulumi convert` panicking on programs containing `Fn::Secret`.
```

Comment:
Strategically putting this above the previous change entry because #264 adds an entry below. 😆

Reply:
Should allow merge w/o conflicts